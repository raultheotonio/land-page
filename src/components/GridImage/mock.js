export default {
  background: false,
  title: 'My Grid Image',
  description: 'Description',
  grid: [
    {
      altText: 'alt text 1',
      srcImg: 'https://source.unsplash.com/random/800x800?r=1',
    },
    {
      altText: 'alt text 2',
      srcImg: 'https://source.unsplash.com/random/800x800?r=2',
    },
    {
      altText: 'alt text 3',
      srcImg: 'https://source.unsplash.com/random/800x800?r=3',
    },
    {
      altText: 'alt text 4',
      srcImg: 'https://source.unsplash.com/random/800x800?r=4',
    },
    {
      altText: 'alt text 5',
      srcImg: 'https://source.unsplash.com/random/800x800?r=5',
    },
  ],
};
