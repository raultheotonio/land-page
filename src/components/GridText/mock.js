export default {
  title: 'title',
  description: 'description',
  background: false,
  grid: [
    {
      title: 'grid 1',
      description: 'grid description',
    },
    {
      title: 'grid 2',
      description: 'grid description',
    },
    {
      title: 'grid 3',
      description: 'grid description',
    },
  ],
};
