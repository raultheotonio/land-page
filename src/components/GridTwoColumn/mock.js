export default {
  title: 'Grid two columns',
  text: 'Somente um teste aqui por favor',
  background: true,
  srcImg: 'assets/images/javascript.svg',
};
