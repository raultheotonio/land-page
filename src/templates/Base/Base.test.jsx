import { Base } from '.';
import renderTheme from '../../utils/RenderTheme';
import { mockBases } from './mock';

describe('<Base />', () => {
  it('should render', () => {
    const { container } = renderTheme(<Base {...mockBases} />);

    expect(container).toMatchSnapshot();
  });
});
