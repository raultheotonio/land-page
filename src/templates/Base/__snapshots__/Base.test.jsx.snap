// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base /> should render 1`] = `
<div>
  <button
    aria-label="Open/Close menu"
    class="sc-gKPRtg gpKKXb"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="StyledIconBase-sc-ea9ulj-0 dXOYdB"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    class="sc-eDvSVe dvWtxw"
  >
    <div
      class="sc-gswNZR DuAtq"
    >
      <div
        class="sc-jSUZER fOjZxU"
      >
        <div>
          <h1
            class="sc-hLBbgP jtiATY"
          >
            <a
              class="sc-iBYQkv nTOzH"
              href="#"
            >
              Logo
            </a>
          </h1>
        </div>
        <nav
          aria-label="Main menu"
          class="sc-ftTHYK ixHmdh"
        />
      </div>
    </div>
  </div>
  <div
    class="sc-bcXHqe imEFwB"
  >
    <div
      class="sc-ksBlkl zTeMC"
    >
      <div
        class="sc-gswNZR DuAtq"
      >
        <div
          class="sc-csuSiG jWrfsW"
        >
          <div>
            <h2
              class="sc-hLBbgP cEbgCx"
            >
              title
            </h2>
          </div>
          <p
            class="sc-jrcTuL QaQrX"
          >
            description
          </p>
          <div
            class="sc-eDWCr jLoRbN"
          >
            <div
              class="sc-bqWxrE iNPcZG"
            >
              <div>
                <h3
                  class="sc-hLBbgP bCIAxa"
                >
                  grid 1
                </h3>
              </div>
              <p
                class="sc-jrcTuL QaQrX"
              >
                grid description
              </p>
            </div>
            <div
              class="sc-bqWxrE iNPcZG"
            >
              <div>
                <h3
                  class="sc-hLBbgP bCIAxa"
                >
                  grid 2
                </h3>
              </div>
              <p
                class="sc-jrcTuL QaQrX"
              >
                grid description
              </p>
            </div>
            <div
              class="sc-bqWxrE iNPcZG"
            >
              <div>
                <h3
                  class="sc-hLBbgP bCIAxa"
                >
                  grid 3
                </h3>
              </div>
              <p
                class="sc-jrcTuL QaQrX"
              >
                grid description
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-kDvujY gRbYms"
    >
      <div
        class="sc-gswNZR DuAtq"
      >
        <p
          class="sc-jrcTuL QaQrX"
        >
          <p>
            Teste de footer
          </p>
        </p>
      </div>
    </div>
  </div>
  <a
    aria-label="Go to top"
    class="sc-ipEyDJ fmeuKO"
    href="#"
    title="go to top"
  >
    <svg
      aria-hidden="true"
      class="StyledIconBase-sc-ea9ulj-0 dXOYdB"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M7.41 15.41 12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
</div>
`;
